db-client-java/src/test/java/com/eventstore/dbclient/SubscribePersistentSubcription.java

```diff
@@ -82,4 +82,59 @@ public void onCancelled(PersistentSubscription subscription) {
 
         Assert.assertEquals(6, result.get().intValue());
     }
+
```

Comment:
**SUGGESTION:** Can we add test scenarios for the negative cases, e.g.:
- what happens if subscribing to a not available database,
- nodes were disconnected during the persistent subscription liveness,
- error while handling the persistent subscription logic,
- etc.

Reply:
Agree, see previous comment.